Vuologist/bumpie
# Email verification

closes #234 

This includes bug fix when navigating from forgot password to sign in

TESTING INSTRUCTIONS
1. Compile and go to /email-verified
2. Check if matches XD
3. Go to /sign-up
4. Click forgot password
5. Then click sign in button 
6. Validate that it goes back to the sign in page
